runtime/src/accounts_hash.rs

```diff
@@ -688,7 +691,6 @@ impl AccountsHash {
         let mut next_pass = PreviousPass::default();
         let cumulative = CumulativeOffsets::from_raw(&hashes);
         let mut hash_total = cumulative.total_count;
-        stats.hash_total += hash_total;
```

Comment:
It isn't obvious, but a few lines above we re-add unprocessed hashes from the previous pass. This results in double-counts of those hashes.
```
hashes.insert(0...```